### DISCUSSION TITLE
Suricata does not alert on some TCP Traffic

### DISCUSSION THREAD

**Discussion by ErwinDFIRocks** [Security-Onion-Solutions/securityonion#3945] [Unsupported Versions]
Hello,

we are facing the issue that Suricata seems to alert normal on ICMP + DNS (mostly small packages at all) e.g. domain requests and stuff like this but not alert on TCP traffic, which is probably larger than e.g. Domain Requests.

We don't see any HTTP/TLS alerts but they are for sure in the traffic/pcap.
If we do replay the captured pcap on the same interface where the original traffic was captured, the alert appears.

We have already checked some stuff in the suricata.yaml (MTU, MSS, QUEUE,...) but nothing seems to resolve the issue.

Anyone an idea?
I would appreciate every hint.

Kind Regards,
Erwin

**Comment by weslambert** [comment]
Are you running just Suricata, or Zeek as well?

Do you noticed anything alarming from Zeek weird logs if running it?
